src/app/(sidebar)/smart-contracts/contract-explorer/components/ContractInfo.tsx

```diff
@@ -276,9 +299,21 @@ export const ContractInfo = ({
           </>
         ),
       },
+      builtIn: {
+        badge: (
+          <Badge variant="success" icon={<Icon.CheckCircle />}>
+            Built-in Contract
+          </Badge>
+        ),
+        message: <>TODO: needs description</>,
```

Comment:
What copy should we show here?
<img width="657" height="199" alt="image" src="https://github.com/user-attachments/assets/2826f8a0-e886-47d1-a112-d049d16ea281" />

Reply:
@quietbits Added the description